modules/aws_mysql/aws-mysql.json

```diff
@@ -29,6 +29,11 @@
       "description": "Add deletion protection to stop accidental db deletions",
       "default": false
     },
+    "db_name": {
+      "type": "string",
+      "description": "The name of the database to create.",
```

Comment:
looks good, the official documentation mentions some restrictions, may be we could also include them -> https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster#database_name

Reply:
If the official documentation already covers this, it would just be a redundancy on our part to do this thing again.

Reply:
the opta users are not exposed to the official terraform documentation, they only see https://docs.opta.dev/reference/aws/modules/aws-mysql/
or did you mean something else?